peppy/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.35.5"
+__version__ = "0.35.6"
```

Comment:
Let's make this 0.40.0 since it's lost backwards compatibility.

Reply:
0.40.0 ? or 0.36.0?

Reply:
0.40.0 since it's lost backwards compatibility.